Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
```

Comment:
### [acts_as_tenant](https://github.com/ErwinM/acts_as_tenant)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.5.1 → 0.5.2

[_~~change-log~~, [source-code](https://github.com/ErwinM/acts_as_tenant), [gem-diff](https://my.diffend.io/gems/acts_as_tenant/0.5.1/0.5.2)_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/ErwinM/acts_as_tenant/compare/v0.5.1...v0.5.2).

These are the first 10 commits:
- (a874288) [Update blog_post.md](https://github.com/ErwinM/acts_as_tenant/commit/a8742885596201feeef7ec6df3a0ae35a3a41bb6)
- (4e94d23) [Merge pull request #268 from axcdnt/patch-1](https://github.com/ErwinM/acts_as_tenant/commit/4e94d23f3965aaaf74e2cc1b709fab0504eecf62)
- (f27385a) [Update README.md](https://github.com/ErwinM/acts_as_tenant/commit/f27385ad1ffe474581679d6bf4464150abf8c033)
- (1be245a) [Merge pull request #269 from basicfeatures/patch-1](https://github.com/ErwinM/acts_as_tenant/commit/1be245a8edda1a5a12471d1cb143cb5489802534)
- (42d02b1) [Removes duplicated error class](https://github.com/ErwinM/acts_as_tenant/commit/42d02b1e8df2ff01bb92355cea45a32d5aeec05f)
- (adfb863) [Merge pull request #272 from TildeWill/remove-duplicated-err…](https://github.com/ErwinM/acts_as_tenant/commit/adfb863c4ef962ccbcc2a06bf54c6545968a0473)
- (de3417e) [Update logic to incorporate changes in ActiveModel::Dirty in…](https://github.com/ErwinM/acts_as_tenant/commit/de3417ea196b8b5c75ee1bc63a996c15cff4d590)
- (619bb45) [Refactor](https://github.com/ErwinM/acts_as_tenant/commit/619bb45092a9040bed6c4568aab910f4519c6603)
- (b978a8b) [Refactor](https://github.com/ErwinM/acts_as_tenant/commit/b978a8bd97de0c3141222c9a268fe79d062e502a)
- (145c0fe) [Add more test case](https://github.com/ErwinM/acts_as_tenant/commit/145c0fed8a21a798bdf2ddc853e9526a134ad0d9)


</details>